user-guide/Basic_Functionality/Visio/SRM/Linking_a_shape_to_a_booking.md

```diff
@@ -78,3 +78,41 @@ The following placeholders can be used in the text displayed on the *Reservation
   This placeholder supports the same options as the *\[Elapsed Time\]* placeholder.
 
 - Placeholders referring to **custom booking properties**, in the format \[*PropertyName*\], e.g. *\[Class\]*.
+
+## Making the booking shape display booking information
+
+From DataMiner 10.3.8/10.4.0 onwards<!-- RN 33215+36489 -->, you can make a shape linked to a booking display all data associated with that booking.
+
+To do so, add a **Component** shape data field and set its value to *BookingData*.
+
+For example:
+
+|Shape data field | Value |
+|-------------|---------------|
+| Reservation | `[pagevar:SRMRESERVATIONS_IDOfSelection]` |
+| Component   | `BookingData` |
+
+The shape will show the following information:
+
+- On the left-hand side, a list of resources used by the booking.
+
+  For every resource, this list shows the following information:
+  
+  - The resource name.
+
+  - An icon indicating the function of the resource.
+
+  - An icon indicating whether the resource is linked to a service definition node.
+
+  - The node label or, if no node label is defined, the name of the function definition.
+
+- On the right-hand side, the profile data of the node or node interface selected in the list on the left:
+
+  - The profile instance (if applicable).
+
+  - The profile parameter values that will be used. Any parameter values defined in the booking override the values defined in the profile instance, which in turn override the values defined in the profile definition.
+
+> [!NOTE]
+>
+> - To use this component, you need to have a Resource Manager and Service Manager license, as well as the user permissions to access the UI of all [SRM modules](xref:SRM).
```

Comment:
Maybe interesting to know which ones?
The 'UI available' of the the following modules: Profiles, Functions, Services, Resources, Bookings.

Reply:
"you need to have a Resource Manager and Service Manager license".
could be redundant: it is also mandatory to have Indexing Engine (elastic DB), but this is always the case when have Service Manager license.